staging/src/k8s.io/apimachinery/pkg/runtime/serializer/json/json_fuzz.go

```diff
@@ -0,0 +1,110 @@
+// Copyright 2019 Google Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License"); you may not
+// use this file except in compliance with the License. You may obtain a copy of
+// the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+// WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+// License for the specific language governing permissions and limitations under
+// the License.
+
+// +build gofuzz
+
+package json
+
+import (
+	"bytes"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+)
+
+func FuzzStrictDecode(data []byte) int {
```

Comment:
Who calls this function? Does the "build gofuzz" tag do some magic?

Reply:
Hi Chao, the intent is that these will run under oss-fuzz continuously.